starter-code/src/functions-and-arrays.js

```diff
@@ -48,6 +97,19 @@ var wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray(array) {
```

Comment:
Nice I like the way you solved this with localCompare(). If you'd like to write it shorter, however, have a look at .filter(), combined with indexOf();